rbme

```diff
@@ -548,19 +550,20 @@ for source in "$@" ; do
 		$RSYNC $RSYNC_EXTRA_OPTIONS --link-dest="$lastbackup/" --relative -aSH \
 			--ignore-errors $rsync_mountpoints_option \
 			--exclude-from=<(
-                    # Exclude typical in-memory stuff by default
-                    cat <<-EOF
-                    /proc/*
-                    /run/*
-                    /sys/*
-                    /dev/shm/*
-                    /dev/pts/*
-                    /**/lost+found/*
-                    /**/.gvfs
-                    EOF
+			# Exclude typical in-memory stuff by default
```

Comment:
So the problem was TAB vs. SPACE? Maybe let's add the missing dirs here, same as we have in ReaR already?